README.md

```diff
@@ -173,89 +173,89 @@ Indy-SDK and Askar store their wallets inside different directories:
 
 It is left to the user to move their wallet(s) into the desired location.
 
-## Step-by-step Acapy Wallet Migration Guide
-0. Stop any agents using the wallet: Before starting the migration process, make sure to stop any agents or applications that are currently using the wallet to avoid any potential conflicts.
-1. Backup your current wallet: It is important to create a backup of your current wallet before starting the migration process, in case anything goes wrong. If using sqlite, copy the indy wallet from `/home/<user>/.indy_client/wallet/<wallet name>` to a temporary location you can run the migration script from. After running migration script you will be coping the resulting db into Askar location.
-2. Prepare configuration: The migration script supports migration from Indy SQLite to Aries SQLite or from Indy PostgreSQL to Aries PostgreSQL. Determine which database and [storage plugin](https://github.com/hyperledger/indy-sdk/tree/main/experimental/plugins/postgres_storage#wallet-management-modes) you are using and gather the necessary information for each scenario. Do note that postgres DBPW is the default wallet_scheme for Indy when using postgres storage plugin.
-wallet migration strategies include `dbpw`, `mwst-as-profiles`, `mwst-as-stores`. Postgres `MWST` wallet scheme serves both MultiWalletSingleTable and MultiWalletSingleTableSharedPool indy storage plugin management styles. The default configuration for ACA-Py multi-tenant agent is managed mode, which will migrate with `mwst-as-profiles` strategy. If you have a shared database for multiple ACA-Py instances but did not use Multi-tenancy you should use `mwst-as-stores` strategy. Here are examples of different strategies with minimum configuration. For MWST schemes you will need to provide a json file that includes the wallet_name, keyed to wallet_key. Example, {<wallet_name>:<wallet_key>,...}.
-    - Indy SQLite -> Aries SQLite:
+## Step-by-step ACA-Py Wallet Migration Guide
+0. Stop any agents using the wallet: Before starting the migration process, make sure to stop any agents or applications that are currently using the wallet to avoid database access conflicts.
+1. Backup your current wallet: It is important to create a backup of your current wallet before starting the migration process, in case anything goes wrong. If using sqlite, copy the indy wallet from `/home/<user>/.indy_client/wallet/<wallet name>` to a temporary location you can run the migration script from. After running migration script you will be coping the resulting db into Askar location. Migrating a postgresql data base will not require any file relocation. Both sqlite and postgresql migration will require updating ACA-Py startup config, which this guide will explain.  
```

Comment:
"After running migration script you will be coping the resulting db into Askar location." is a bit confusing the first time I read it.  I think you will be running the script later, but the verbiage seems to indicate otherwise if you haven't read the whole guide yet.
I suggest something like "After running the migration script (which you will do in just a minute) you will be copying the resulting db into the Askar location."